Makefile

```diff
@@ -230,30 +245,30 @@ ifeq ($(MINIKUBE_BUILD_IN_DOCKER),y)
 	$(call DOCKER,$(BUILD_IMAGE),/usr/bin/make $@)
 else
 	which go-bindata || GO111MODULE=off GOBIN=$(GOPATH)/bin go get github.com/jteeuwen/go-bindata/...
-	PATH="$(PATH):$(GOPATH)/bin" go-bindata -nomemcopy -o pkg/minikube/translate/translations.go -pkg translate translations/...
+	PATH="$(PATH):$(GOPATH)/bin" go-bindata -nomemcopy -o $@ -pkg translate translations/...
 	-gofmt -s -w $@
 endif
 	@#golint: Json should be JSON (compat sed)
 	@sed -i -e 's/Json/JSON/' $@ && rm -f ./-e
 
 .PHONY: cross
-cross: out/minikube-linux-$(GOARCH) out/minikube-darwin-amd64 out/minikube-windows-amd64.exe
+cross: minikube-linux-amd64 minikube-darwin-amd64 minikube-windows-amd64.exe
 
 .PHONY: windows
-windows: out/minikube-windows-amd64.exe
+windows: minikube-windows-amd64.exe
```

Comment:
Why not in out/?

Reply:
Oh, it is still created in out/. Just using the new (shorter) targets.

Reply:
And the odd GOARCH's are still available, just not built by "cross".